src/modules/75-cf/pages/feature-flags/components/__tests__/FlagTableFilters.test.tsx

```diff
@@ -31,37 +54,54 @@ describe('FlagTableFilters', () => {
     })
   })
 
+  afterAll(() => {
+    jest.resetAllMocks()
+  })
+
   test('FlagTableFilters should render correctly the filters for feature flags', async () => {
-    render(
-      <TestWrapper
-        path="/account/:accountId/cf/orgs/:orgIdentifier/projects/:projectIdentifier/feature-flags"
-        pathParams={{ accountId: 'dummy', orgIdentifier: 'dummy', projectIdentifier: 'dummy' }}
-      >
-        <FlagTableFilters features={mockFeatureFlags as any} currentFilter={{}} updateTableFilter={jest.fn()} />
-      </TestWrapper>
-    )
+    renderComponent()
+
+    const { featureCounts } = mockFeatureFlags
+
+    await waitFor(() => {
+      const filterCards = document.getElementsByClassName('Card--card')
+      expect(filterCards).toHaveLength(6)
+      expect(filterCards[0].getElementsByTagName('p')[0].textContent).toBe('cf.flagFilters.allFlags')
+      expect(filterCards[0].getElementsByTagName('p')[1].textContent).toBe(featureCounts.totalFeatures.toString())
+      expect(filterCards[0].closest('.Card--card')).toHaveClass('Card--selected')
+      expect(filterCards[1].getElementsByTagName('p')[0].textContent).toBe('cf.flagFilters.enabled')
+      expect(filterCards[1].getElementsByTagName('p')[1].textContent).toBe(featureCounts.totalEnabled.toString())
+      expect(filterCards[2].getElementsByTagName('p')[0].textContent).toBe('cf.flagFilters.permanent')
+      expect(filterCards[2].getElementsByTagName('p')[1].textContent).toBe(featureCounts.totalPermanent.toString())
+      expect(filterCards[3].getElementsByTagName('p')[0].textContent).toBe('cf.flagFilters.last24')
+      expect(filterCards[3].getElementsByTagName('p')[1].textContent).toBe(
+        featureCounts.totalRecentlyAccessed.toString()
+      )
+      expect(filterCards[4].getElementsByTagName('p')[0].textContent).toBe('cf.flagFilters.active')
+      expect(filterCards[4].getElementsByTagName('p')[1].textContent).toBe(featureCounts.totalActive.toString())
```

Comment:
I think if you use `toEqual` instead of `toBe`, you can avoid the `toString()` call.
```suggestion
      expect(filterCards[4].getElementsByTagName('p')[1].textContent).toEqual(featureCounts.totalActive)
```